lab_2_keywords_cooccurrence/main.py

```diff
@@ -48,13 +132,15 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    if not type_check(phrases, list) or not type_check(stop_words, list):
+
+    if not (my_isinstance(phrases, list) and my_isinstance(stop_words, list)):
         return None
-    candidates = []
-    for phrase in [phrase.lower().split() for phrase in phrases]:
-        splits = [-1] + [index for index, word in enumerate(phrase) if word in stop_words] + [len(phrase)]
-        candidates.extend(tuple(candidate) for start, end in pairwise(splits) if (candidate := phrase[start+1:end]))
-    return candidates
+
+    text_splited = " † ".join(phrases).lower().split()
+    stop_words_destroyed = " ".join([word if word not in stop_words else "†" for word in text_splited])
+    key_phrases_empty_items = [tuple(phrase.split()) for phrase in re.split("†", stop_words_destroyed)]
+    key_phrases = list(filter(lambda x: x != tuple(), key_phrases_empty_items))
```

Comment:
Do not use checks like `== tuple()` or `!= list()`

Reply:
solved